Tasks/DeployAzureResourceGroup/AzureUtilityLTE9.8.ps1

```diff
@@ -37,15 +37,18 @@ function Deploy-AzureResourceGroup
           [string]$overrideParameters)
 
     Switch-AzureMode AzureResourceManager
-    Write-Host "[Azure Resource Manager]Creating resource group deployment with name $resourceGroupName"
+
+    $deploymentName = [System.IO.Path]::GetFileNameWithoutExtension($csmFile) + '-' + ((Get-Date).ToUniversalTime()).ToString('yyyyMMdd-HHmm')
+
+    Write-Host "[Azure Resource Manager]Creating resource group deployment with name $deploymentName"
```

Comment:
Have $resourceGroupName here instead of the $deploymentName. Additionally it would be even better if you could print something like: "Creating resource group with name: $resourceGroupName and deployment name: $deploymentName".